src/styles/main.scss

```diff
@@ -1,3 +1,208 @@
-body {
-  margin: 0;
+$color-main: #060b35;
+$color-secondary: #616070;
+$border-radius: 5px;
+$border-card: 1px solid #f3f3f3;
+$blue-color: #00acdc;
+$white-color: #fff;
+$button-border-hover: 1px solid #00acdc;
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul {
+  margin: 0 auto;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background: $white-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+
+  &__nav-list {
+    display: flex;
+    padding: 0;
+    text-transform: uppercase;
+    list-style: none;
+  }
+
+  &__nav-item {
+    display: flex;
+  }
+
+  &__nav-item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__nav-link {
+    position: relative;
+    color: $color-main;
+    text-decoration: none;
+    padding: 23px 0;
+  }
+
+  &__logo {
+    left: 50px;
```

Comment:
The 'left' property is not needed for the '.header__logo' class. Consider removing it to avoid potential positioning issues.